pulp_smash/utils.py

```diff
@@ -0,0 +1,9 @@
+# coding=utf-8
+"""Utility functions for Pulp tests."""
+from __future__ import unicode_literals
+from uuid import uuid4
+
+
+def rand_str():
```

Comment:
Despite the definition of this new function, the coverage statistics don't change. That's because the `make test-coverage` make target needs to be tweaked. That said, this module is enormously simple, so I'm not terribly worried about fixing this right away. We can leave this as is, but if we do so, a GitHub issue should be created to fix this issue.